src/foam/nanos/dig/ExternalDataAware.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @license
+ * Copyright 2021 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.INTERFACE({
+  package: 'foam.nanos.dig',
+  name: 'ExternalDataAware',
+  documentation: 'Interface for marking an object that contains the externalData map',
+
+  methods: [
+    {
+      name: 'getExternalData',
+      type: 'java.util.Map'
```

Comment:
We have 'Map' type.   did that not work here?  
Our Map is designed for both Java and javascript.